Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ruby:2.6.6-buster
+
+ARG user=sk
```

Comment:
preco potrebujeme nastavit pouzivatela?

Reply:
Pytas sa, ze preco to davame ako build argument resp. preto vobec to bezat pod non-root usrom? 

Ak prve, tak uplne nutne to nie je, je to len kvoli variabilite, nech si to pripadne vie clovek zmenit. 
Ak druhe, tak myslim, ze je to lepsie z dovodov: 1. [best practices a security](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user) 2. je sanca, ze ak je uid:gid tvojho usra na laptope rovnake, ake ma user v kontajneri (1000:1000), tak nemusis riesit problemy s permissions, resp. ak v kontajnery vytvorit subory, tak na ne budes vediet pristupit aj z laptopu/hostu. Ale samozrejme nie je to one-size-fits-all riesenie.